cookbook/upgrade/_update_dep_errors.rst.inc

```diff
@@ -0,0 +1,19 @@
+Dependency Errors
+~~~~~~~~~~~~~~~~~
+
+If you get a dependency error, it may simply mean that you need to upgrade
+other Symfony dependencies too. In that case, try the following command:
+
+    $ composer update symfony/symfony --with-dependencies
+
+This updates ``symfony/symfony`` and *all* packages that it depends on,
+will include several other packages. By using tight version constraints in
```

Comment:
The second part of the first sentences doesn't sound quite right. Maybe something like "which will include several other packages. [...]" is better?